.java-version

```diff
@@ -0,0 +1 @@
+15
```

Comment:
leci will build this using Java 8 so the jar we published can be used as dependency in all projects that use java version >= 8

Reply:
so set it to 8?

Reply:
yes, so we are sure it builds with 8